docs/whats-new.md

```diff
@@ -61,17 +65,17 @@ A standardized set of light classes are now supported by multiple material model
 
 ### Modularization Improvements
 
-- Modularization of luma.gl is also continuing and in some cases rarely used code has been removed to reduce library bloat.
+- luma.gl has been restructured to make it easier for applications to select and import only parts of the library that they use.
 
 * ` @luma.gl/gpgpu` **<sup>New</sup>** - A new experimental submodule with GPGPU Algorithms and Utilities has been added, containing a growing collection of GPU accelerated algorithms and utility methods.
 
 
 ### Performance Instrumentation
 
-Extensive metrics about frame CPU and GPU times, resource counts, and GPU memory usage are being collected. The data is exposed as a proble.gl `Stats` object. The new probe.gl [`StatsWidget`]() can be used to present data in applications.
+Extensive metrics about frame CPU and GPU times, resource counts, and GPU memory usage are being collected. The data is exposed as a [probe.gl](https://uber-web.github.io/probe.gl/#/) `Stats` object. The new probe.gl StatsWidget can be used to present data in applications.
```

Comment:
I think you could get a link directly to the doc page for the stats object. assuming we published one...